toolkit/tools/pkg/imagecustomizerlib/customizebootloader.go

```diff
@@ -30,12 +30,12 @@ var (
 	ErrBootloaderUnsupportedRootMountId = NewImageCustomizerError("Bootloader:UnsupportedRootMountId", "unsupported root mount identifier")
 )
 
-func handleBootLoader(ctx context.Context, rc *ResolvedConfig, imageConnection *imageconnection.ImageConnection,
+func handleBootLoader(ctx context.Context, buildDir string, rc *ResolvedConfig, imageConnection *imageconnection.ImageConnection,
```

Comment:
`buildDir` is available in `ResolvedConfig`.

Reply:
Oh, nice, thanks.